CoreLibrary/src/CoreLib/CORERobot.h

```diff
@@ -38,6 +48,9 @@ class CORERobot {
 	void teleopInit(void);
 	void teleop(void);
 	void teleopEnd(void);
+	void addMotor(motors motors,CANSpeedController* motor);

```

Comment:
For example, "motors motors" is needlessly confusing and hard for people to parse. should at least be changed to "motors motor", and probably "Motor motor, CANSpeedController* controller".